Firestore/tests/System/TransactionTest.php

```diff
@@ -237,4 +237,27 @@ public function testAbort()
         $this->assertTrue($didRetry);
         $this->assertEquals('bat', $this->document->snapshot()['foo']);
     }
+
+    public function getAggregateCases()
+    {
+        // This dataProvider returns the test cases to test
+        // how aggregations work with in a transaction.
+        //
+        // The values are of the form
+        // [
+        //      $aggregationType,
+        //      $arg,
+        //      $expectedResult,
+        //      $docsToAddBeforeTestRunning
+        // ]
```

Comment:
This is a more pythonic way of adding comments in a function. Can we move this to a doc block above the function?

Reply:
Same for other places

Reply:
_I wonder what is the equivalent of pythonic in PHP 😛_

Anyways, I've fixed that in the commit: https://github.com/googleapis/google-cloud-php/pull/6717/commits/053ddde3eefd299839536308ccd2b22ba944a22d